src/ui/components/play-export-dialog.ts

```diff
@@ -14,53 +14,122 @@ declare global {
 @customElement('play-export-dialog')
 export class PlayExportDialog extends LitElement {
   static override styles = css`
-    .close {
-      float: right;
-    }
     dialog {
-      background-color: var(--color-interactive-background);
+      color: var(--color-neutral-content);
+      background-color: var(--color-neutral-background);
+
       border-bottom-left-radius: var(--radius);
       border-bottom-right-radius: var(--radius);
       border-top-left-radius: var(--radius);
       border-top-right-radius: var(--radius);
-      border-width: var(--border-width);
-      border-color: var(--color-neutral-border);
 
-      /* Add gap between viewport. */
-      margin-top: var(--space);
+      padding-top: var(--space);
+      padding-bottom: var(--space);
+      padding-left: var(--space);
+      padding-right: var(--space);
 
-      /* h2 has enough spacing. */
-      padding-top: 0;
+      /* RPL/Shadow/Medium */
```

Comment:
:no_bell: Do we use this in multiple places? If so, consider a CSS variable like `--shadow-10: 0px 4px 8px 0px var(--color-shade-10), 0px 6px 12px 0px var(--color-shade-25)` or something.